src/imitation/algorithms/adversarial/common.py

```diff
@@ -29,7 +29,7 @@ def compute_train_stats(
 
     Args:
         disc_logits_gen_is_high: discriminator logits produced by
-            `DiscrimNet.logits_gen_is_high`.
+            `algorithms.adversarial.common.AdversarialTrainer.logits_gen_is_high`.
```

Comment:
Do we need the fully qualified name? AdversarialTrainer is defined in this very file.